includes/GbWeiss.php

```diff
@@ -138,9 +140,43 @@ public function validateCredentials(): void
         }
     }
 
+    /**
+     *  Requests a new OAuthToken and stores the accessToken in
+     *  the ws_options table
+     *
+     *  @throws \Exception If the token could not be saved.
+     */
+    public function updateToken(): bool
+    {
+        $clientId = get_option('gbw_client_id', false);
+        $clientSecret = get_option('gbw_client_secret', false);
+
+        try {
+            $token = $this->authenticationClient->getToken($clientId, $clientSecret);
+            if ($token->isValid()) {
```

Comment:
This check is not necessary since the getting the token will always return a valid token.